src/components/templates/PartyMap/PartyMap.tsx

```diff
@@ -23,6 +24,7 @@ export const PartyMap: React.FC = () => {
   useConnectCurrentVenue();
   const { user, profile } = useUser();
   const { recentVenueUsers } = useRecentVenueUsers();
+  const [, , isCustomSoundsLoaded] = useCustomSound();
```

Comment:
Since we're returning multiple objects here now, it would make sense for us to move from returning an array, to returning an object of named keys like we do in most of our other custom hooks.

Reply:
Even with the changes described below (about waiting for the loading in the `CustomSoundsProvider` component), it would probably be good to make these changes still + have this variable exposed from the hook (even if we don't end up needing to use it here in our code)

Reply:
Strongly agree, added 
```
type UseCustomSoundType = {
  play: PlaySpriteFunction,
  data: ExposedDataWithPlay,
  isLoaded: boolean,
};
```
as returned type from `useCustomSound`